app/controllers/pieces_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
   def update
     @piece = Piece.find(params[:id])
     @game = Game.find(@piece.game_id)
-    @piece.update_attributes(piece_params)
+    @piece.move_to!(piece_params[:row_position].to_i, piece_params[:col_position].to_i)
```

Comment:
I don't think we're using it right now. But will we use when a pawn is promoted?